recipes/libxml2/all/conanfile.py

```diff
@@ -171,6 +170,9 @@ def _build_msvc(self):
             args.append(f"lib=\"{';'.join(libdirs)}\"")
 
             for name in self._configure_option_names:
+                # FIXME: can't `del` "run-debug" options
+                if Version(self.version) >= "2.11.3" and name == "run-debug":
+                    continue
```

Comment:
What do you mean by `can't del "run-debug" options`? You can remove with rm_safe it dash is the problem. Could you elaborate whether this option still make sense in autotools branch? What about mingw branch?